### PR TITLE
Add ChangeLightsColor and ResetLightsColor 

### PR DESCRIPTION
**Pull request by ClaudioPanConQueso**

### CODE REVIEW

**Comment by ClaudioPanConQueso** [comment]
will i get bullied again?

**Review by louis1706 — APPROVED** [review]

File: Exiled.API/Features/Map.cs [hunk 1]
```diff
@@ -217,6 +217,25 @@ public static void TurnOffAllLights(float duration, IEnumerable<ZoneType> zoneTy
...
+                light.NetworkOverrideColor = color;
+        }
+
+        /// <summary>
+        /// Resets the <see cref="Color">color</see> of all lights in the facility.
+        /// </summary>
+        public static void ResetLightsColor()
+        {
+            foreach (RoomLightController light in RoomLightController.Instances)
+                light.NetworkOverrideColor = Color.clear;
```
Comment (Misaka-ZeroTwo) [inline Exiled.API/Features/Map.cs:236]: I recall Color.clear is not the default color in game. Could you double-check it?

Comment (ClaudioPanConQueso) [reply Exiled.API/Features/Map.cs:236]: actually, it is

Comment (ClaudioPanConQueso) [reply Exiled.API/Features/Map.cs:236]: here's the Room.ResetColor()

        public void ResetColor()
        {
            Color = Color.clear;
        }

Comment (Misaka-ZeroTwo) [reply Exiled.API/Features/Map.cs:236]: My bad, that was another thing.

**Review by Misaka-ZeroTwo — APPROVED** [review]

**Pull request merged**